bot/cogs/cyber.py

```diff
@@ -368,6 +372,17 @@ def suffix_from_number(num):
             text += "We’re currently allocating students to their preferred locations so it’s an ongoing process!"
             text += " We’ll send out details of your location as soon as we can. It shouldn’t be too long!"
             await message.channel.send(text)
+        # Level Function Regex.
```

Comment:
Consider adding a blank line here too?

```suggestion

        # Level Function Regex.
```

Reply:
will do rn